src/compute/src/sink/materialized_view_v2.rs

```diff
@@ -639,6 +645,7 @@ mod write {
     pub fn render<S>(
```

Comment:
Do you mind adding documentation to `render`? It's argument list is long enough for me to lose context when reading it.

Reply:
I added a third commit that adds documentation for the parameters passed to the render methods.